### ISSUE TITLE
fail if file for include cannot be found

### ISSUE THREAD

**Issue by robstoll** [liquibase/liquibase#1102]
## Environment

**Liquibase Version**: 3.8.9

**Liquibase Integration & Version**: maven

**Liquibase Extension(s) & Version**:  none

**Database Vendor & Version**: mariadb

**Operating System Type & Version**: ubuntu 19.10

## Description
```
databaseChangeLog:
  - include:
      - file: nonExistingFile.yml
```
should fail with an error

## Steps To Reproduce

see above, run it

## Actual Behavior

it silently keeps on as the include command never happened

## Expected/Desired Behavior
It errors instead

**Comment by molivasdat** [comment]
Hi @robstoll Thanks again for submitting the issue.  For my understanding, if the  file does not exist and there is not a file extension, liquibase will not error but just continue as if the file existed but there is no warning or error that it did not process the nonexistent file.  If there is any extension on the file, and it does not actually exist, liquibase will error and say that the file does not exist.

**Comment by robstoll** [comment]
Exactly, I have updated the description to put more emphasize on the inconsistency.

**Comment by nvoxland** [comment]
People may be relying on the existing behavior, so this will have to add a a flag to include to control it it. Probably best to use `errorIfMissingOrEmpty` like includeAll has.

The change to the XSD means this will have to go out in a major release, not a patch release.

**Comment by robstoll** [comment]
@nvoxland which behaviour do you mean? that a non-existing file without extension is ignored? This one I heavily doubt as the behaviour is the same as if one does not include it at all and one could also just comment something out if required.

**Comment by molivasdat** [comment]
Hi @nvoxland Gentle reminder on this one.

**Comment by kotoant** [comment]
Double checked on the latest versions:
- 3.10.3
- 4.4.3
The issue is still reproduced.

Please could you fix it (including 3.x.x version)?

Thanks

**Comment by filipelautert** [comment]
Hey, this was resolved back in PR #3901 (merged June 2023) which added `errorIfMissingOrEmpty` support to `include` and `property` elements — defaulting to `true`, so missing files now error as expected.

Closing this one out. If you're still seeing issues on a recent version, feel free to re-open.
